addons/auth_totp/data/mail_templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_totp_invite" model="mail.template">
```

Comment:
Guidelines: file should be mail_template_data.xml :D (as it hold mail.template records). mail_templates.xml is used for QWeb templates though.